sbol_utilities/component.py

```diff
@@ -519,3 +519,141 @@ def ed_protein(definition: str, **kwargs) -> sbol3.ExternallyDefined:
     :return: A Component object.
     """
     return sbol3.ExternallyDefined([sbol3.SBO_PROTEIN], definition, **kwargs)
+
+def ed_restriction_enzyme(definition: str, name:str, **kwargs) -> sbol3.ExternallyDefined:
+    """Creates an ExternallyDefined Restriction Enzyme Component.
+
+    :param definition: The URI that links to a canonical definition external to SBOL, recommended UniProt.
```

Comment:
If you want to have this be standard REBASE enzymes, leave definition out entirely, and construct it from the name.

Reply:
implemented using `definition=f'http://rebase.neb.com/rebase/enz/{name}.html'`